lib/src/reactive_ble.dart

```diff
@@ -285,8 +284,8 @@ class FlutterReactiveBle {
   /// Use [servicesWithCharacteristicsToDiscover] to scan only for the specific services mentioned in this map,
   /// this can improve the connection speed on iOS since no full service discovery will be executed. On Android
   /// this variable is ignored since partial discovery is not possible.
-  /// The [connectionTimeout] parameter can be used to emit a failure after a certain period in case the connection
-  /// is not established. The pending connection attempt will be cancelled.
+  /// If [connectionTimeout] parameter is supplied and a connection is not established before [connectionTimeout] expires,
+  /// the pending connection attempt will be cancelled and a [TimeoutException] error will be emitted into the returned stream.
```

Comment:
Can you also add the auto connect description here?

- [ ] Add auto connect description